biz.aQute.resolve/build.gradle

```diff
@@ -0,0 +1,21 @@
+
+test {
```

Comment:
Using the task name directly is wrong for our build. We need to maintain task configuration avoidance.

`tasks.named('test') {`